src/main/java/com/amazonaws/kinesisvideo/java/client/KinesisVideoJavaClientFactory.java

```diff
@@ -101,7 +100,8 @@ public static KinesisVideoClient createKinesisVideoClient(
         Preconditions.checkNotNull(deviceInfo);
         Preconditions.checkNotNull(executor);
 
-        final Log log = new Log(configuration.getLogChannel(), LogLevel.DEBUG, "KinesisVideo");
+        final Log log = Log.getLogInstance("KinesisVideo");
+        log.setCurrentLogLevel(Level.DEBUG);
```

Comment:
does this work at a different level than the one in log4j.xml ? cant we get the logger context and use that level?